api/server.js

```diff
@@ -20,18 +21,19 @@ manifestPromise.then(manifest => {
     server.auth.strategy('oidc_session', 'oidc_session');
     server.auth.strategy('email_token', 'email_token', {
       findToken: async id => {
-        let token = await bookshelf.model('email_token')
-          .forge({ token: id })
-          .where('expires_at', '>', bookshelf.knex.fn.now())
-          .fetch();
-
-        if (!token) {
-          token = await bookshelf.model('user_password_reset_token')
-            .forge({ token })
-            .where('expires_at', '>', bookshelf.knex.fn.now())
+        try {
```

Comment:
Doing a `try { } catch()` with nothing in the catch block except `throw error` is the same as not having a `try { } catch()` block. You should just remove the try/catch stuff